tests/run/alpha/Test_2.java

```diff
@@ -0,0 +1,12 @@
+
+package alpha;
+
+public class Test_2 {
+
+  public static void main(String[] args) {
+    Alpha<String> a = new Bar();
+    assert a.foo() == 1;
+    assert a.bar("a").equals("aa");
+    Alpha<String> aa = a.append(a);
```

Comment:
I'd also like to see a test where Java extends a class with `@alpha` methods and overrides them.
More complicated, what happens when Scala with `@alpha` overrides Java methods.

Even more tricky: what about:
```scala
// Scala
class A {
  @alpha("bar") def foo(): Int = 1
}
```
```java
// Java
public class B extends A {
  @Override
  public int bar() {
    return 2;
  }
}
```
```scala
// Scala
class C extends B {
  // is this correct?
  @alpha("bar") override def foo(): Int = 3

  // what about this?
  override def bar(): Int = 4
}
```

Reply:
A Scala method with @alpha cannot possibly override a Java method. Overriding before and after erasure must be the same. I explore three possible scenarios in a new test neg/alpha-override.